docker/entrypoint.sh

```diff
@@ -17,12 +18,12 @@ COMMAND="${CMD_NAME}"
 
 echo "Linting commits:"
 git config --global --add safe.directory "$PWD"
-git log --color=always --graph --oneline HEAD~..HEAD^2 || true
+git log --color=always --graph --oneline "${commits}" || true
 echo ""
 echo "Against 'committed.toml':"
 ${COMMAND} --dump-config - || true
 echo ""
 echo "If this fails, don't sweat it. We're trying to encourage clear communication and not hinder contributions."
 echo "If it is a reasonable issue and you lack time or feel uncomfortable fixing it yourself,"
 echo "let us know and we can mentor or fix it."
-${COMMAND} --color=always -vv --no-merge-commit HEAD~..HEAD^2
+${COMMAND} --color=always -vv "${commits}"
```

Comment:
Why was `--no-merge-commit` removed?  How do we plan to handle that case?

Reply:
Could you instead swap it around to not be breaking change?

Reply:
Good point! Done in 8452a4248c4910fe599b9b72c6989a7508171084